src/style.css

```
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  height: 300px;
+  flex-basis: 100%;
+  color: rgb(255,255,255);
```

make it simple as possible
```suggestion
  color: white;
```